src/login/login.component.tsx

```diff
@@ -106,26 +108,36 @@ export default function Login(props: LoginProps) {
                 cursor: pointer;
               `}
             >
-              <svg
-                className="omrs-icon"
-                role="img"
-                css={css`
-                  top: calc(50% - 0.75rem);
-                  right: 0.75rem;
-                  position: absolute;
-                `}
-              >
+              <svg className="omrs-icon" role="img">
                 <use xlinkHref="#omrs-icon-visibility" />
               </svg>
             </button>
           </div>
+          <div
+            css={css`
+              text-align: center;
+            `}
+          >
+            <p
+              css={css`
+                color: var(--omrs-color-danger);
+              `}
+            >
+              {errorMessage}
+            </p>
+          </div>
           <div>
             <button
-              className="omrs-btn omrs-filled-action"
+              className={`omrs-margin-top-24 omrs-btn ${
```

Comment:
Kremling's toggle function would make this look nicer:
```jsx
<button className={always("omrs-margin-top-24 omrs-btn omrs-btn-lg").toggle("omrs-filled-disabled", "omrs-filled-action", !password || !username)}
```

No need to change though, just sharing information.